samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -168,7 +185,7 @@ The sample supports the following development kit:
 
 .. table-from-rows:: /includes/sample_board_rows.txt
    :header: heading
-   :rows: nrf5340dk_nrf5340_cpunet
+   :rows: nrf5340dk_nrf5340_cpunet, nrf21540dk_nrf52840
```

Comment:
Add a line for the added board to the sample_board_rows.txt file. It does not seem to be there yet.

Reply:
Added